packages/oasis-actions/src/strategies/aave/close/close.ts

```diff
@@ -288,42 +304,74 @@ async function buildOperation(
     .integerValue(BigNumber.ROUND_DOWN)
 
   const fee = feeResolver(args.collateralToken.symbol, args.debtToken.symbol)
-
-  const closeArgs = {
-    // In the close to collateral scenario we need to add the preSwapFee amount to the fromTokenAmount
-    // So, that when taking the fee from the source token we are sending the Swap contract
-    // the sum of the fee and the ultimately fromAmount that will be swapped
-    lockedCollateralAmountInWei: args.shouldCloseToCollateral
-      ? swapData.fromTokenAmount.plus(swapData.preSwapFee)
-      : args.collateralAmountLockedInProtocolInWei,
-    flashloanAmount: amountToFlashloanInWei,
-    fee: fee.toNumber(),
-    swapData: swapData.exchangeCalldata,
-    receiveAtLeast: swapData.minToTokenAmount,
-    proxy: dependencies.proxy,
-    collectFeeFrom: swapData.collectFeeFrom,
-    collateralTokenAddress,
-    collateralIsEth: args.collateralToken.symbol === 'ETH',
-    debtTokenAddress,
-    debtTokenIsEth: args.debtToken.symbol === 'ETH',
-    isDPMProxy: dependencies.isDPMProxy,
-    shouldCloseToCollateral: args.shouldCloseToCollateral || false,
+  const lockedCollateralAmountInWei = args.shouldCloseToCollateral
```

Comment:
If I understand correctly that is the amount that is going to be swapped. I think the naming is very misleading.  The approach is good but the name could be  something like `swappableCollateralAmount` or `collateralAmountToBeSwapped` or  `swapAmount` or something like that.  The locked collateral amount is what we currently have in the position. After close it will be always 0 no matter what you are closing to. You simply won't have any collateral locked, right ?